static/styles/settings.css

```diff
@@ -248,10 +248,15 @@ select.setting_email_notifications_batching_period_seconds {
 
 #uploaded_files_table > tr > td:nth-of-type(1),
 .upload-file-name {
-    width: 30%;
+    width: 43%;
     word-break: break-all;
 }
 
+.upload-mentioned-in,
+.active {
```

Comment:
I think would be better to use a more specific selector instead of `.active`. I don't think `.active` is used currently for other element, but since we are doing it to properly break the sorting arrow, seems would be better to make selector specific.

Reply:
Sure, changing  `.active` to `.upload-date`